### PR TITLE
[11.8] Add reviewer_id and wip fields to "merge_request -> all" route

### PR DESCRIPTION
**Pull request by jkniest**
Hello there 👋 

I've added the fields `reviewer_id` and `wip` as described in the documentation: https://docs.gitlab.com/ee/api/merge_requests.html to the merge request route.

I wasn't sure about the changelog, so I didn't touch it for now. Should I add an entry to it?

Thanks, and have a nice day!

### CODE REVIEW

File: src/Api/MergeRequests.php [hunk 1]
```diff
@@ -61,6 +61,8 @@ class MergeRequests extends AbstractApi
      *     @var string             $labels         return merge requests matching a comma separated list of labels
      *     @var \DateTimeInterface $created_after  return merge requests created after the given time (inclusive)
      *     @var \DateTimeInterface $created_before return merge requests created before the given time (inclusive)
+     *     @var int                $reviewer_id    returns merge requests which have the user as a reviewer with the given user id
+     *     @var string             $wip            return only draft merge requests (yes) or only non-draft merge requests (no)
```
Comment (GrahamCampbell) [inline src/Api/MergeRequests.php]: I think this should be a boolean, and be transformed in the method. Can you change this please?

**Review by GrahamCampbell — CHANGES_REQUESTED** [review]

**Comment by GrahamCampbell** [comment]
Thanks for the PR. I have left an inline comment. :)

File: src/Api/MergeRequests.php (hunk 1 above)
Comment (jkniest) [reply src/Api/MergeRequests.php]: Heyho! I've changed it. Hope, this is okay, if not let me know :)

Thanks!

**Review by GrahamCampbell — APPROVED** [review]

**Comment by GrahamCampbell** [comment]
Thanks. This will be included in the v11.8.0 release, probably next month.

**Pull request merged**